### PR TITLE
feat: SDK-189 Switch to new SDK configuration

### PR DESCRIPTION
**Pull request by Max-Stytch**
Updates the style and view configuration to the new standard established in [SDK Themes](https://docs.google.com/document/d/19slwoKdJ_AkOBFTgIHgJPClXKZk0RuPYjnokcbXg-aM/edit#)

### CODE REVIEW

File: public/signupOrLogin.html [hunk 1]
```diff
@@ -43,29 +43,22 @@
...
+          loginExpirationMinutes:30,
+          signupRedirectURL: "http://localhost:9000/authenticate",
+          signupExpirationMinutes: 30,
+        }
+      };
+      stytch.mount({
         elementId: "#magic-link",
         style: style,
-        config: loginOrCreateUserConfig
+        loginOrSignupView: loginOrCreateUserConfig
```
Comment (Max-Stytch) [inline public/signupOrLogin.html:61]: Is there a better variable name we can use than `loginOrCreateUserConfig`?

Comment (jeremy-stytch) [reply public/signupOrLogin.html:61]: `magicLinkDefaults`?

**Review by jeremy-stytch — APPROVED** [review]
LGTM, but I think this should get a review from someone more familiar with the SDK. Assigning @nathan-stytch

**Review by nathan-stytch — APPROVED** [review]

File: public/signupOrLogin.html (hunk 1 above)
Comment (Max-Stytch) [reply public/signupOrLogin.html]: Addressed in https://github.com/stytchauth/stytchjs-html-express/pull/18/commits/02e2301923095f3e2e3c5816b7871121e08ddc73

**Pull request merged**